aldryn_newsblog/cms_toolbar.py

```diff
@@ -26,8 +27,10 @@ class NewsBlogToolbar(CMSToolbar):
     watch_models = [Article, ]
     supported_apps = ('aldryn_newsblog',)
 
-    def get_on_delete_redirect_url(self, article):
-        url = reverse('{0}:article-list'.format(article.app_config.namespace))
+    def get_on_delete_redirect_url(self, article, language=None):
```

Comment:
@czpython, you're right. We are calling this only inside this class, and context manager will work even with language being `None`, so better to explicitly require language.